src/util/routes.js

```diff
@@ -3,6 +3,8 @@ import { getZoneName } from "./domain";
 
 const MAX_WIDTH = 6;
 const MAX_HEIGHT = 25;
+// The value to divide a pixel height by to reach the unit used for MAX_HEIGHT.
+const STOP_PX_HEIGHT = 35;
```

Comment:
MAX_HEIGHT is how many stops can be shown in a branch of the route diagram and it uses a bespoke unit. This is an approximation of what I think was used to divide a pixel value to reach the MAX_HEIGHT value. Correct me if I'm wrong...